openbas-api/src/main/java/io/openbas/config/AppConfig.java

```diff
@@ -49,4 +56,20 @@ public OpenAPI openBASOpenAPI() {
                 .description("OpenBAS documentation")
                 .url("https://docs.openbas.io/"));
   }
+
+  @PostConstruct
```

Comment:
Why not having that in a dedicated file with @Configuration annotation ?

Reply:
Good point